implementation/src/main/java/io/smallrye/config/inject/SecuritySupport.java

```diff
@@ -24,11 +24,11 @@
 /**
  * @author <a href="http://jmesnil.net/">Jeff Mesnil</a> (c) 2018 Red Hat inc.
  */
-class SecuritySupport {
+public class SecuritySupport {
```

Comment:
I don't think this class is intended to be public. Probably needs to be copied to another package if it's needed, or moved.

@dmlloyd ??

Reply:
This definitely should *not* be public, as that will cause a CVE.

Reply:
Ah sorry, I changed that when I was refactoring and forgot to go back and investigate what should really be done with it. Looks like there is a `SecuritySupport` class in the package it's moved to and it should be using that one.